app/models/policy.rb

```diff
@@ -49,10 +49,11 @@ def fill_from(profile:)
   def update_hosts(new_host_ids)
     return unless new_host_ids
 
-    policy_hosts.where.not(host_id: new_host_ids).destroy_all
-    PolicyHost.import((new_host_ids - host_ids).map do |host_id|
+    removed = policy_hosts.where.not(host_id: new_host_ids).destroy_all
+    imported = PolicyHost.import((new_host_ids - host_ids).map do |host_id|
       { host_id: host_id, policy_id: id }
     end)
+    [imported.ids.count, removed.count]
```

Comment:
`num_inserts` on the import Result seems to return number of INSERT statements as it did return 1 for two imported items. `ids` should work for Postgres.